content/en/infrastructure/cloud_cost_management.md

```diff
@@ -85,7 +95,23 @@ Attach the new S3 policy to the Datadog integration role.
 3. Click **Attach policies**.
 4. Enter the name of the S3 bucket policy created above.
 5. Click **Attach policy**.
+
+## Investigating your costs
+
+Navigate to your costs within Datadog by clicking **Infrastructure > Cloud Costs** in the UI. This view lets you quickly filter and group your spending to hone in on what’s changing over time, answering questions such as:
+
+- Which teams and applications are responsible for changes in spend?
+- What cloud products and usage types (i.e. EC2 Data Transfer) are driving the increase in spend?
+- How effective is the coverage and utilization of my discount plans?
```

Comment:
Could we maybe add examples here?
Something like this:
Which teams and applications are responsible for changes in spend? Try grouping by service and team.

Also, not so sure about discount plans. How are we doing this?

Reply:
We have the foundation set up for discount plan coverage/utilization, but we are not exposing this yet